Detectors/MUON/MCH/Mapping/test/CMakeLists.txt

```diff
@@ -59,6 +59,7 @@ o2_add_test(StressTest3
         COMMAND_LINE_ARGS --testpos ${CMAKE_CURRENT_LIST_DIR}/data/test_random_pos.json --run2 --manunumbering
         PUBLIC_LINK_LIBRARIES O2::MCHMappingImpl3 O2::MCHMappingSegContour
         RapidJSON::RapidJSON
+        CONFIGURATIONS RelWithDebInfo Release MinSizeRel
```

Comment:
I like this. My only point is, could the check be inverse, i.e. `NOT Debug`, instead of explicitly listing the allowed configurations?

Reply:
@davidrohr I'm afraid the add_test command (which is the one in the end implementing this CONFIGURATIONS option, I'm just forwarding fro o2_add_test to it) does not allow that unfortunately. I tried just in case to convince myself I was correctly reading the CMake doc, but what is generated is not what we'd like : 

```
[ (python) brew ]~/alice/brew/sw/BUILD/O2-latest/O2$ rg CTEST_ -t cmake
Detectors/MUON/MCH/Mapping/test/CTestTestfile.cmake
11:if("${CTEST_CONFIGURATION_TYPE}" MATCHES "^([Nn][Oo][Tt]|[Dd][Ee][Bb][Uu][Gg])$")
```